docs/github-secrets.md

```diff
@@ -11,15 +11,12 @@ The same secret described there, should be deployed to the `build-service` and
 
 ```bash
 kubectl -n pipelines-as-code create secret generic pipelines-as-code-secret \
-...
 ```
 
 ```bash
 kubectl -n build-service create secret generic pipelines-as-code-secret \
-...
 ```
 
 ```bash
 kubectl -n integration-service create secret generic pipelines-as-code-secret \
-...
```

Comment:
The commands here should not be used as-is (removing the `...` won't make those commands work).

The actual command is part of the pipelines-as-code documentation that is linked above, and the idea was that the user repeats the command that they just did for namespace `pipelines-as-code` also for the other two namespaces.

Any suggestions for what could have conveyed this intention better?

(We try to copy as little as possible from external docs so we don't have to maintain their changes as well in our docs)

Reply:
Well, it actually works with removing the dots.  tried. 
Also, it has a copy command on the side hence the intention.

Reply:
Ok, not sure what is the source of confusion. my suggestion is to keep everything as is and just delete the 3 points at the end of the command as suggested. I tested it and it is working.

Reply:
Do you mean you run this command?
```
kubectl -n build-service create secret generic pipelines-as-code-secret \
```
If so, this will keep the prompt open (as the lines ends with `\`), and if you press enter again, it will just create an empty secret, which will not return an error, but also will not allow the different controllers to talk with github.

Reply:
oh, got it ... so maybe just remove the command from command box with copy so user will know there is no intent for him to run it?

Reply:
If we can make it look decent and it will make things clearer then I'm fine with that.